dandiapi/api/tests/factories.py

```diff
@@ -76,6 +77,19 @@ class PublishedVersionFactory(BaseVersionFactory):
     class Meta:
         model = Version
 
+    @classmethod
+    def _create(cls, *args, **kwargs):
+        version: Version = super()._create(*args, **kwargs)
+        version.doi = f'10.80507/dandi.{version.dandiset.identifier}/{version.version}'
```

Comment:
Where does that hardcoded DOI prefix come from? Should it be parameterized somehow?

I realize this is a testing module, so maybe it doesn't matter here, but if this value corresponds to a production datacite instance or something like that, I wonder if it should be stored in and retrieved from an environment variable or something like that.

Reply:
https://github.com/dandi/dandi-api/issues/472